### ISSUE TITLE
write_ppm

### ISSUE THREAD

**Issue by erenfn** [karansher/computer-graphics-raster-images#83]
Should write_ppm deal with duplicate files?

**Comment by abhimadan** [comment]
No, don't worry about that. If the file exists, and you write to the file, it should just overwrite the old file.
